WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2284,3 +2284,22 @@ WhiteBlackListViewRowCell                  = font-weight:normal;
 WhiteBlackListRemoveItemCell               = width:@BigIconWidth@; @ListItemIconWrapper@
 WhiteBlackListViewItemsNumberUsed          = margin-top:@WhiteBlackViewInnerSpacing@;
 
+######################################
+#   OptionsItems ListView
+######################################
+
+OptionsItemsListViewItemsInnerSpacing      = 10px;
+OptionsItemsListColumnHeaderWrapper        = border-left:0px;
+OptionsItemsListColumnHeader               = padding-left:@OptionsItemsListViewItemsInnerSpacing@;
+OptionsItemsListRow                        = overflow:hidden;
+OptionsItemsListLastRow                    = border-bottom:none;
+OptionsItemsListRows                       = overflow:visible;
+OptionsItemsListRowCell                    = padding-left:@OptionsItemsListViewItemsInnerSpacing@; text-align:left;
+OptionsItemsListRowCellSelected            = @OptionsItemsListRowCell@; line-height:1.7rem; white-space:normal;
```

Comment:
line height value is needed here.
Removed OptionsItemsListRowCell . Thanks !
https://github.com/Zimbra/zm-web-client/pull/98/commits/6a6a14dd2f05f961c3d5d8d962c046d40c19d080